grails-app/services/au/org/ala/biocollect/merit/ProjectService.groovy

```diff
@@ -909,7 +909,8 @@ class ProjectService {
                         index = facets.size()
                         facets.add(index, specialFacet.clone())
                     } else {
-                        facets.putAt(index, specialFacet.clone())
+                        int facetIndex = facets.size()
```

Comment:
Any reason why this code has changed? It is supposed to replace special facets at the index found in `hubFacets`.